### PR TITLE
LAMBDA, Lesson 1 revised

### PR DESCRIPTION
**Pull request by grosu**
I'm starting to revise the tutorial lecture notes in parallel with you guys fixing bugs and adding missing features.  These should be really easy to review.  For now, @cos suffices to approve them, but at some moment it would be a good idea if @laurayuwen and @daejunpark also take a look.

### CODE REVIEW

File: k-distribution/tutorial/1_k/1_lambda/lesson_1/NOTES.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+We now support the following line to the syntax module:
+
+  syntax priority lambda_._ > __  // exact syntax subject to change
```
Comment (cos) [inline k-distribution/tutorial/1_k/1_lambda/lesson_1/NOTES.md]: Would look better wrapped in code quotes. Cannot edit it myself on your fork.

File: k-distribution/tutorial/1_k/1_lambda/lesson_1/README.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+[MOVIE]
 
-Here we define our first K module, which contains the initial syntax of LAMBDA,
-and learn how to use the basic K commands.
+Here we define our first K module, which contains the initial syntax of the
+LAMBDA language, and learn how to use the basic K commands.
 
-I have just created an empty working folder, and opened a terminal window
+We have just created an empty working folder, and opened a terminal window
 (to the left) and an editor window (to the right).  We will edit our K
```
Comment (cos) [inline k-distribution/tutorial/1_k/1_lambda/lesson_1/README.md:11]: Would it be better to use present tense?

**Comment by cos** [comment]
@dwightguth, I think the mvn checkstyle doesn't know we're in 2016. Do you know where to change that by any chance?

**Comment by grosu** [comment]
@cos, check again and merge when done.  thanks.

**Comment by cos** [comment]
will merge after #2002 (I should have made a 14 more issues for the pull request to be #2016 :)

File: k-distribution/tutorial/1_k/1_lambda/lesson_1/NOTES.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+We now support the following line to the syntax module:
```
Comment (cos) [inline k-distribution/tutorial/1_k/1_lambda/lesson_1/NOTES.md:3]: needs copyright line

Comment (grosu) [inline k-distribution/tutorial/1_k/1_lambda/lesson_1/NOTES.md:3]: Done

**Comment by cos** [comment]
Jenkins: test this please

**Pull request merged**